tests/integration/test_create.py

```diff
@@ -40,7 +46,8 @@ def test_cluster_memory(client, c, df):
             new_table
         WITH (
             location = 'df',
-            format = 'memory'
+            format = 'memory',
+            gpu = {gpu}
```

Comment:
Good catch - it looks like regardless of the `gpu` kwarg here, the created table is a Pandas dataframe. If we're okay with having failing gpuCI tests for now, I can add the `to_pandas` call now and we can sort out this failure in a follow up PR

Reply:
Sounds good, we can add the to_pandas call and mark as `xfail` and we can sort out the error in a followup. 

Could you also target the pr to the new 21.12 branch?

Reply:
After digging into this, it looks like this is because when we call `LocationInputPlugin.to_dc(format="memory")`, it just returns the exact dataframe that was published to the default client (which for this test would always be a Dask dataframe):

https://github.com/rapidsai/dask-sql/blob/4d5f7dd4c133f7f2c584db2492c8f0ef3cba7e31/dask_sql/input_utils/location.py#L18-L29

I'll xfail and open a PR for to resolve this